src/test/java/com/microsoft/sqlserver/jdbc/bulkCopy/BulkCopyISQLServerBulkRecordTest.java

```diff
@@ -89,7 +89,7 @@ public void testISQLServerBulkRecord() throws SQLException {
     @Test
     public void testBulkCopyDateTimePrecision() throws SQLException {
         String dstTable = TestUtils
-                .escapeSingleQuotes(AbstractSQLGenerator.escapeIdentifier(RandomUtil.getIdentifier("dstTable")));
+                .escapeSingleQuotes(AbstractSQLGenerator.escapeIdentifier(RandomUtil.getIdentifier("dstTableBulkCopyDt")));

```

Comment:
None of the previous code here was leaving objects in the DB, but the generic name "dstTable" was used elsewhere, making it difficult to quickly identify the offending test.